addons/crm/models/crm_lead.py

```diff
@@ -604,6 +599,7 @@ def write(self, vals):
         stage_updated, stage_is_won = vals.get('stage_id'), False
         # stage change: update date_last_stage_update
         if stage_updated:
+            vals['date_last_stage_update'] = fields.Datetime.now()
```

Comment:
I don't get it, if we update it any time we write to the stage how is it different from the compute? Won't the same thing happen?

Reply:
Hello @thiryr ,
When we change the user_id, we will only get team_id and user_id in vals, not stage_id. The stage_id will get in vals only when the stage is changed. But in compute whenever user_id changes it will also update date_last_stage_update because stage_id depends on team_id.

Thank you!

Reply:
But doesn't the compute call write? I guess maybe not actually :thinking: 

Regardless, not sure if you saw my review comment but a test would be good for this :)

Reply:
Compute call _write I think. Sometimes it is effectively easier to hook create / write than writing computed fields.